docs/contributing/style-guide.md

```diff
@@ -1,7 +1,405 @@
+Tabs on this site is for code-specific descriptions or examples, since many such things on our platform
+are relevant to multiple languages.
+Ideally these examples will exist in Java, Node.js, Python, Ruby, C#, and the tabs should be placed in that order.
+
+To use tabs, you need to import two special methods by placing these lines below the [Frontmatter](#frontmatter),
+but above the [Introduction](#introduction):
+```markdown
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+### Code Example:
+```markdown
+<Tabs
+  defaultValue="java"
+  values={[
+    {label: 'Java', value: 'java'},
+    {label: 'Node.js', value: 'js'},
+  ]}>
+ 
+<TabItem value="java">
+ 
+This would include information or examples for Java
+
+</TabItem>
+<TabItem value="js">
+
+This would include information or examples for JavaScript
+ 
+</TabItem>
+</Tabs>
+```
+
+### Display
+
+<Tabs
+defaultValue="java"
+values={[
+{label: 'Java', value: 'java'},
+{label: 'Node.js', value: 'js'},
+]}>
+
+<TabItem value="java">
+
+This would include information or examples for Java
+
+</TabItem>
+<TabItem value="js">
+
+This would include information or examples for JavaScript
+
+</TabItem>
+</Tabs>
+
+## Inline Code
+
+To refer to a single class or method name within a sentence, place single backticks around the name.
+This code:
+```markdown
+This comment refers to the `RemoteWebDriver` class
+```
+will be displayed as:
+This comment refers to the `RemoteWebDriver` class
+
+## Code Blocks
+
+The best way to display code is with code blocks. Markdown will highlight each language differently, so it is helpful
+to specify which language you are using, and it's a good idea to include a title with the code block as well.
+
+### Example Code
+
+    ```java title="Custom Title"
+    // Java Code Example
+    ```
+
+### Display
+
+```java title="Custom Title"
+// Java Code Example
+```
+
+## Code References
+
+The Sauce Labs Open Source Team created a plugin for use with Docusaurus to allow us to reference code on Github
+rather than duplicating code in this repo. Ideally all code displayed in the Sauce Labs documentation points to code in
+one of the `demo-<language>` repos on [Sauce Labs Training Github Org](https://github.com/saucelabs-training/?q=demo#org-repositories)
+When referencing code, Include the language, "reference" and a title indicating what the code is.
+The URL for the link can be for the entire file, or include specific line numbers at the end.
+Additionally, the referenced code should reference a tag name rather than just a branch name (which can easily break) or
+a commit hash (which is hard to version). For example, the `demo-java` repo has a tag for `website-1.0`, so the reference
+link should specify this.
+
+### Example Code
+
+    ```java reference title="Example Test"
+    https://github.com/saucelabs-training/demo-java/blob/website-1.0/selenium-examples/src/test/java/com/saucedemo/junit5/SauceBindingsTest.java#L35-L43
+    ```
```

Comment:
```suggestion
```java reference title="Example Test"
https://github.com/saucelabs-training/demo-java/blob/website-1.0/selenium-examples/src/test/java/com/saucedemo/junit5/SauceBindingsTest.java#L35-L43
```
```

Reply:
This one actually needs to be indented so that the tick marks display on the page, otherwise it treats it as a code block, which we don't want.